src/cosmosdb-preview/azext_cosmosdb_preview/tests/latest/test_cosmosdb_mongocluster_scenario.py

```diff
@@ -0,0 +1,131 @@
+            'rg_new': resource_group_new,
+            'loc': 'eastus',
+            'admin_user': self.create_random_name(prefix='cli', length=8),
+            'admin_password': 'Cli1@asvrct',
+            'server_version': '5.0',
+            'shard_node_tier': 'M40',
+            'shard_node_tier_update': 'M50',
+            'shard_node_ha': True,
+            'shard_node_disk_size_gb': 128,
+            'shard_node_count': 2,
+        })
+
+        # Create Cluster
+        cluster = self.cmd('az cosmosdb mongocluster create --cluster-name {c} --resource-group {rg} --location {loc} --administrator-login {admin_user} --administrator-login-password {admin_password} --server-version {server_version} --shard-node-tier {shard_node_tier} --shard-node-ha {shard_node_ha} --shard-node-disk-size-gb {shard_node_disk_size_gb} --shard-node-count {shard_node_count}').get_output_in_json()
+        assert cluster['provisioningState'] == 'Succeeded'
+
+        # show cluster 
+        cluster = self.cmd('az cosmosdb mongocluster  show -c {c} -g {rg}').get_output_in_json()
+        assert cluster['provisioningState'] == 'Succeeded'
+
+        # list cluster
+        clusters = self.cmd('az cosmosdb mongocluster list -g {rg}').get_output_in_json()
+        assert len(clusters) == 1
+    
+        # create resource group
+        self.cmd('az group create -l {loc} -n {rg_new}')
+        
+        cluster = self.cmd('az cosmosdb mongocluster create --cluster-name {c_new} --resource-group {rg_new} --location {loc} --administrator-login {admin_user} --administrator-login-password {admin_password} --server-version {server_version} --shard-node-tier {shard_node_tier} --shard-node-ha {shard_node_ha} --shard-node-disk-size-gb {shard_node_disk_size_gb} --shard-node-count {shard_node_count}').get_output_in_json()
+        assert cluster['provisioningState'] == 'Succeeded'
+
+        clusters = self.cmd('az cosmosdb mongocluster list -g {rg_new}').get_output_in_json()
+        assert len(clusters) == 1
+
+        # update sku
+        cluster = self.cmd('az cosmosdb mongocluster update --cluster-name {c_new} --resource-group {rg_new} --administrator-login {admin_user} --administrator-login-password {admin_password} --server-version {server_version} --shard-node-tier {shard_node_tier_update} --shard-node-ha {shard_node_ha} --shard-node-disk-size-gb {shard_node_disk_size_gb}').get_output_in_json()
+        assert cluster['provisioningState'] == 'Succeeded'
+        
+        # delete non existent cluster, NoContent response
+        try: 
+            delete = self.cmd('az cosmosdb mongocluster delete -c blah -g {rg} --yes')
+            assert delete==None
+        except Exception as e:
+            print(e)
+
+        # Delete Clusters
+        try:
+            self.cmd('az cosmosdb mongocluster delete -c {c} -g {rg} --yes')
+            self.cmd('az cosmosdb mongocluster delete -c {c_new} -g {rg_new} --yes')
+            self.cmd('az group delete -n {rg_new}')
```

Comment:
The resource group will be auto deleted which is implemented in `ResourceGroupPreparer`,
you don't have to do it manually

Reply:
rg_new is a resource group created as another rg in the test, so deleting this explicitly. Can you please double check if this will also be deleted.